client/src/pages/Home.js

```diff
@@ -2,26 +2,27 @@ import Navbar from "./components/Navbar";
 import Footer from "./components/Footer";
 import FavesCard from "./components/FavesCard";
 import WhatsNewCard from "./components/WhatsNewCard";
+
 import "./Home.css";
 
 export function Home() {
 	return (
-	<main className="main-page">
```

Comment:
Just a general thought - have you agreed a code style and you're therefore changing the white space (on this and other pages). 

The white space should remain consistency so that each person commiting code is not changing it back and forth and generating unnecessary changes.

Reply:
yes we did , its the style that related to every page on this App

Reply:
I'm not commenting on the CSS - I'm talking about the white space change that is generating this whole diff.

Reply:
I can edit the css if u are agree with
As we agreed in team once we had a meeting

Reply:
I'm not talking about the CSS. I'm only talking about white space. If you don't understand just ignore, it doesn't matter for the code for now it's just about how to effectively manage PRs.

Reply:
I have not thought about the white space issue but understand why it could be a problem , I have also remade my code after taking on board Azin's suggestions which were valid, and that might have contributed to the white space issue I'll try to keep that in mind in the future